__tests__/app.test.js

```diff
@@ -151,4 +151,37 @@ describe('/api/articles/:article_id/comments', () => {
             expect(response.body.msg).toBe('no comments found');
         });
     });
+})
+
+describe('POST /api/articles/:article_id/comments', () => {
+    test('POST:201 responds with an object of the posted comment ', () => {
+        const postComment = {
+            username: 'lurker',
+            body: 'this is body text'
+          };
+        return request(app)
+        .post('/api/articles/1/comments')
+        .send(postComment)
+        .expect(201)
+        .then(({body}) => {
+            expect(body.comment.comment_id).toBe(19);
+            expect(body.comment.body).toBe('this is body text');
+            expect(body.comment.article_id).toBe(1);
+            expect(body.comment.author).toBe('lurker');
+            expect(body.comment.votes).toBe(0);
+        })
```

Comment:
Would also be good to check for created_at, (any string would be acceptable, since it will change depending on the time of the test)